public/img/illustrations/icons/icon.svg

```diff
@@ -0,0 +1,12 @@
+<svg width="48" height="48" viewBox="0 0 48 48" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
I think it would make sense to compress all those SVG files with https://github.com/svg/svgo. 
For example, this SVG in particular can reduce its size by more than half. We talk about bytes, but there are more graphics to load on our website, and in the end, compressing them is free.

Reply:
Fine, I will do it.